PatientRegistrationApp/src/App.vue

```diff
@@ -164,7 +164,7 @@
 									<v-toolbar flat>
 										<v-toolbar-title>QR Code Page</v-toolbar-title>
 									</v-toolbar>
-									<v-card flat><HouseholdFollowUp/></v-card>
+									<v-card flat><HouseholdFollowUp v-bind:numberOfHouseholdMembers="getNumberOfHouseholdMembers()"/></v-card>
```

Comment:
Not sure where this occurs in the code, but I would remove this final page as part of the pages counted in how many we have left. It's not really a page that needs to be completed so I don't think it should be part of the count here. This should ideally occur in the single patient workflow as well. 

I'm fine putting this work into another card also!

![image](https://user-images.githubusercontent.com/67079030/103323350-0fc9a880-4a08-11eb-8505-586c1b7a231e.png)

Reply:
On that note, I'd honestly remove this UI indicator from this page all together. It's no longer needed since there aren't more pages after this.

Reply:
The way the patient registration app is set up is through the stepper (which is the page counter on the top of the page). Therefore, in order to connect pages and proceed when a "Next" button is clcked is through the stepper. We can add another card to "investigate" the possibility of excluding this page from the stepper, but I believe that this was the most reasonable way to do it (for not at least).

As for the lline of code that addresses the number of household members, it is needed to present however how many members is there and their QR codes as well. 

Did I address all your questions? And how would you like me to proceed?

Reply:
For me at least I think having the indicator there at this point reassures the patient that they have completed all the steps. I also know that we have a pretty tight schedule to stick to, and based off what Shahd brought up it might be a bit complicated to resolve. If this is something that we want to modify, I think it would be best to discuss with the team but I'm not sure how relevant this change would be for our demo.

Reply:
I understand the logic need of possibly needing to keep the bar, but I figure we can keep the stepper implemented while still getting rid of it. Regardless, this change isn't for the demo specifically (none of them usually are). Don't think we need to bring this up with the team since it seems to be a bigger change. 

@ShahdSawalhi Don't worry about this for now

@grohde1015 I disagree that this is an indicator that they made it. I think the final "Submit" and "are you sure you want to continue" buttons clearly indicate that you're done and hitting this page implies your info properly processed.